yarn.lock

```diff
@@ -7783,7 +7783,12 @@ typedoc@^0.17.4:
     shelljs "^0.8.3"
     typedoc-default-themes "^0.10.0"
 
-typescript@^3.8.3, typescript@~3.8.3:
+typescript@^3.9.2:
+  version "3.9.2"
+  resolved "https://registry.yarnpkg.com/typescript/-/typescript-3.9.2.tgz#64e9c8e9be6ea583c54607677dd4680a1cf35db9"
+  integrity sha512-q2ktq4n/uLuNNShyayit+DTobV2ApPEo/6so68JaD5ojvc/6GClBipedB9zNWYxRSAlZXAe405Rlijzl6qDiSw==
+
+typescript@~3.8.3:
```

Comment:
Why do we still have the old typescript? b/c of typedoc?

Reply:
@ninok `ts-json-schema-generator` requires `~3.8.3`, namely doesn't allow minor upgrade.